custom/icds_reports/utils/aggregation_helpers/distributed/birth_preparedness_forms.py

```diff
@@ -116,7 +116,7 @@ def aggregation_query(self):
             GREATEST(ucr.counsel_accessible_ppfp, prev_month.counsel_accessible_ppfp) as counsel_accessible_ppfp,
             COALESCE(ucr.ifa_last_seven_days, prev_month.ifa_last_seven_days) as ifa_last_seven_days,
             COALESCE(ucr.reason_no_ifa, prev_month.reason_no_ifa) as reason_no_ifa,
-            GREATEST(ucr.new_ifa_tablets_total, prev_month.new_ifa_tablets_total) as new_ifa_tablets_total,
+            COALESCE(ucr.new_ifa_tablets_total, prev_month.new_ifa_tablets_total) as new_ifa_tablets_total,
```

Comment:
Should this default to `0` like the other if it is null

Reply:
I am not able to comment on it, but I think the `ucr` part of this is wrong for the same reason the other one is wrong